### PR TITLE
Multiple changes

### PR DESCRIPTION
**Pull request by cochi2**
* Add MediaStreamingConfiguration to CreateCallOptions.
* Changing 'callbackUri' name to 'callbackUrl' for AnswerCall.
* Changing URI type for String on 'callbackUrl' for CreateCallOptions.
* Removing redundant code.

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java [hunk 1]
```diff
@@ -66,32 +66,32 @@ public Response<CreateCallResult> createCallWithResponse(CreateCallOptions creat
      * Answer an incoming call
      *
      * @param incomingCallContext The incoming call context.
-     * @param callbackUri The call back uri. Optional
+     * @param callbackUrl The call back uri. Optional
```
Comment (juntuchen-msft) [inline sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java:69]: Please help remove the "Optional" word just like the one in Async client. Also change the uri to url please

File: sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java [hunk 2]
```diff
@@ -66,32 +66,32 @@ public Response<CreateCallResult> createCallWithResponse(CreateCallOptions creat
...
+    public AnswerCallResult answerCall(String incomingCallContext, String callbackUrl) {
+        return callAutomationAsyncClient.answerCall(incomingCallContext, callbackUrl).block();
     }
 
     /**
      * Create a call connection request from a source identity to a target identity.
      *
      * @param incomingCallContext The incoming call context.
-     * @param callbackUri The call back uri. Optional
+     * @param callbackUrl The call back uri. Optional
```
Comment (juntuchen-msft) [inline sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java:83]: Same as above

**Review by juntuchen-msft — CHANGES_REQUESTED** [review]
Just some minor change required in the comment

File: sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java (hunk 1 above)
Comment (cochi2) [reply sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java]: Removed the `optional` wording.
The parameter was already `String`

**Review by juntuchen-msft — APPROVED** [review]

**Review by bharat-kalyan-namburi — APPROVED** [review]

**Pull request merged**